test/DateFn.mocha.js

```diff
@@ -154,6 +154,11 @@ describe('#DateFn', function () {
         }]
         assert.deepStrictEqual(fixResult(res), exp)
       })
+      it('No Tevet 1 in gregorian year 2024', function () {
```

Comment:
are you sure that this one test covers all the production code updated in this PR?